### PR TITLE
Improve/format

### PR DESCRIPTION
**Pull request by zrgt**
- Fixed bib links
  - bib links were specified, when the was a single adoc document. I defined proper xref links for it referencing sources from bibliography.adoc
- Added image and clause ref links
  - Set descriptive names and ids to images
  - Assigned block IDs for refs
  - Created ref links
- Removed unnecessary spaces in some titles
- Add appendix tables to [.appendix-table] css class
- Renamed adoc files according to antora recommendations
  - Removed uppercases in file names acc. to Antora docs: "Although not required, we strongly recommend that you always use lowercase for filenames. Some filesystems are case sensitive, while others are not. By always using lowercase, you avoid any problems that occur if the filesystem, webserver, or transfer tool does not preserve the casing." (https://docs.antora.org/antora/latest/organize-content-files/#storing-your-content-source-files)
  - Removed "IDTA-01005" Prefix in filenames, as already present in parent directory name

### CODE REVIEW

**Comment by zrgt** [comment]
You can see the generated docs here: https://rwth-iat.github.io/aas-specs-antora/IDTA-01005/test/index.html

File: documentation/IDTA-01005/antora.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: IDTA-01005
 title: DRAFT - Specification Asset Administration Shell - Part 5 AASX Package Format
-version: 'snapshot'
+version: 'test'
```
Comment (BirgitBoss) [inline documentation/IDTA-01005/antora.yml:3]: I prefer snapshot

File: documentation/IDTA-01005/modules/ROOT/nav.adoc [hunk 2]
```diff
@@ -10,13 +10,13 @@ Illustrations:
...
-** xref:./Annex/IDTA-01005_Background.adoc[Background]
-* xref:IDTA-01005_ChangeLog.adoc[Change Log]
-* xref:IDTA-01005_Bibliography.adoc[Bibliography]
+* xref:introduction.adoc[Introduction]
+* xref:aasx.adoc[Specification]
+* xref:summary-and-outlook.adoc[Summary and Outlook]
+* xref:./annex/background.adoc[Annex]
+** xref:./annex/background.adoc[Background]
+* xref:changelog.adoc[Change Log]
+* xref:bibliography.adoc[Bibliography]
```
Comment (BirgitBoss) [inline documentation/IDTA-01005/modules/ROOT/nav.adoc:19]: why did you change it?  the file names start (and shall start) with IDTA-01005

File: documentation/IDTA-01005/modules/ROOT/pages/annex/background.adoc [hunk 3]
```diff
@@ -22,6 +22,6 @@ The Open Packaging Conventionsfootnote:[Not to be confused with OPC (Open Platfo
...
-* Office Open XML (including the Open Packaging Conventions format) and Open Document Format are politically conflicting formats (see details in link:#bib2[[2\]] and link:#bib3[[3\]]). Choosing Open Packaging Conventions as the option for storing the Asset Administration Shell information had only technical reasons based on the arguments mentioned here.
+* The Office Open XML and Open Packaging Conventions specifications originated from the Microsoft Corporation and were later standardized as ISO/IEC 29500 and ECMA-376. Current and future versions of ISO/IEC 29500 and ECMA-376 are covered by Microsoft's Open Specification Promise, whereby Microsoft "irrevocably promises" not to assert any claims against those making, using, and selling conforming implementations of any specification covered by the promise (so long as those accepting the promise refrain from suing Microsoft for patent infringement in relation to Microsoft's implementation of the covered specification) xref:bibliography.adoc#bib1[[1\]].
```
Comment (BirgitBoss) [inline documentation/IDTA-01005/modules/ROOT/pages/annex/background.adoc:25]: see above: file name are not correct, start with IDTA-01005

File: documentation/IDTA-01005/modules/ROOT/pages/annex/background.adoc [hunk 4]
```diff
@@ -22,6 +22,6 @@ The Open Packaging Conventionsfootnote:[Not to be confused with OPC (Open Platfo
...
+* Office Open XML (including the Open Packaging Conventions format) and Open Document Format are politically conflicting formats (see details in xref:bibliography.adoc#bib2[[2\]] and xref:bibliography.adoc#bib3[[3\]]). Choosing Open Packaging Conventions as the option for storing the Asset Administration Shell information had only technical reasons based on the arguments mentioned here.
```
Comment (BirgitBoss) [inline documentation/IDTA-01005/modules/ROOT/pages/annex/background.adoc:26]: see above: file names

File: documentation/IDTA-01005/modules/ROOT/pages/summary-and-outlook.adoc [hunk 5]
```diff
@@ -12,7 +12,7 @@ SPDX-License-Identifier: CC-BY-4.0
 
 This document (Part 5 of the document series "Details of the Asset Administration Shell") defines the AASX package file format for file exchange of Administration Shells based on the Open Package Conventions.
 
-Additional parts of the document series cover (see link:#bib10[[10\]]):
+Additional parts of the document series cover (see xref:bibliography.adoc#bib10[[10\]]):
```
Comment (BirgitBoss) [inline documentation/IDTA-01005/modules/ROOT/pages/summary-and-outlook.adoc:15]: see above: file names

**Review by BirgitBoss — CHANGES_REQUESTED** [review]
Please chekc file names: they start with IDTA-01005 and this is also how it is done in Part 1, Part 2 etc.

File: documentation/IDTA-01005/antora.yml (hunk 1 above)
Comment (zrgt) [reply documentation/IDTA-01005/antora.yml:3]: I've changed the version name to generate an additional version in antora, so you can see the generated documentation. We will set it back to snapshot after the commit was approved.

**Comment by zrgt** [comment]
> Please chekc file names: they start with IDTA-01005 and this is also how it is done in Part 1, Part 2 etc.

@BirgitBoss I have noticed that the .adoc files are in a folder called "IDTA-01005", which indicates that all the files belong to "IDTA-01005" and it is already part of the path and therefore redundant. This means that when creating documents with Antora, paths such as "https://rwth-iat.github.io/aas-specs-antora/IDTA-01005/snapshot/IDTA-01005_AASX.html" are created.

I have therefore removed the prefix from all file names. Another reason for this adjustment is that the cross-references between the files become unnecessarily long. It would be worth considering removing these superfluous prefixes in order to streamline the structure and improve the readability of the references, as the affiliation to the document "IDTA-01005" is already established by the location of files in "IDTA-01005" folder.

If you are ok with it, we can adapt names of .adoc files also in Part 1, 2 etc. This is done fast. If not, we can revert the renaming commit here.

File: documentation/IDTA-01005/antora.yml (hunk 1 above)
Comment (BirgitBoss) [reply documentation/IDTA-01005/antora.yml:3]: or "in progress" like discussed today

File: documentation/IDTA-01005/modules/ROOT/nav.adoc [hunk 6]
```diff
@@ -10,13 +10,13 @@ Illustrations:
...
-* xref:IDTA-01005_AASX.adoc[Specification]
-* xref:IDTA-01005_SummaryOutlook.adoc[Summary and Outlook]
-* xref:./Annex/nav_annex.adoc[Annex]
-** xref:./Annex/IDTA-01005_Background.adoc[Background]
-* xref:IDTA-01005_ChangeLog.adoc[Change Log]
-* xref:IDTA-01005_Bibliography.adoc[Bibliography]
+* xref:introduction.adoc[Introduction]
+* xref:aasx.adoc[Specification]
+* xref:summary-and-outlook.adoc[Summary and Outlook]
+* xref:./annex/background.adoc[Annex]
```
Comment (BirgitBoss) [inline documentation/IDTA-01005/modules/ROOT/nav.adoc]:
```suggestion
* xref:./annex/nav_annex.adoc[Annex]
```

File: documentation/IDTA-01005/modules/ROOT/nav.adoc (hunk 2 above)
Comment (BirgitBoss) [reply documentation/IDTA-01005/modules/ROOT/nav.adoc:19]: decision today: files without prefix

File: documentation/IDTA-01005/modules/ROOT/nav.adoc (hunk 6 above)
Comment (BirgitBoss) [reply documentation/IDTA-01005/modules/ROOT/nav.adoc]: Both ways are possible but I am not sure it is a good idea to let the Annex start with the first Annex part. Of course nav_annex could include also some text, e.g. like in structure of the document
